lumigator/schemas/lumigator_schemas/jobs.py

```diff
@@ -148,6 +148,19 @@ class JobResults(BaseModel):
     artifact_url: str
 
 
+class JobResultObject(BaseModel):
+    """This is a very loose definition of what data
+    should be stored in the output settings.S3_JOB_RESULTS_FILENAME.
+    As long as a job result file only has the fields defined here,
+    it should be accepted by the backend.
+    """
+
+    model_config = ConfigDict(extra="forbid")
+    metrics: dict | None
+    parameters: dict | None
+    artifacts: dict | None
```

Comment:
Maybe dict[str, Any] to a be a bit more, er, pydantic?
Also, I'd rather use an empty dict as the default value instead of allowing None. I don't think there's any difference in semantics, but this allows the user of these fields to handle them in an homogeneous way.

Reply:
Good point. I was trying to make the result object as flexible as possible but maybe I went too far 😆 . I'll update.

Reply:
Note that your proposal is not strictly bad!!!! I was just trying to come up with something with better properties, and these were only suggestions.